internal/distributed/proxy/httpserver/utils.go

```diff
@@ -128,12 +127,14 @@ func printFields(fields []*schemapb.FieldSchema) []gin.H {
 	var res []gin.H
 	for _, field := range fields {
 		fieldDetail := gin.H{
-			HTTPReturnFieldName:       field.Name,
-			HTTPReturnFieldPrimaryKey: field.IsPrimaryKey,
-			HTTPReturnFieldAutoID:     field.AutoID,
-			HTTPReturnDescription:     field.Description,
+			HTTPReturnFieldName:         field.Name,
+			HTTPReturnFieldPrimaryKey:   field.IsPrimaryKey,
+			HTTPReturnFieldPartitionKey: field.IsPartitionKey,
+			HTTPReturnFieldAutoID:       field.AutoID,
+			HTTPReturnDescription:       field.Description,
 		}
-		if field.DataType == schemapb.DataType_BinaryVector || field.DataType == schemapb.DataType_FloatVector {
+		if field.DataType == schemapb.DataType_BinaryVector || field.DataType == schemapb.DataType_FloatVector ||
```

Comment:
use this function

milvus/pkg/util/typeutil/schema.go

![image](https://github.com/milvus-io/milvus/assets/59249785/0d861af6-ac50-47c3-85fc-3612ac8b66ee)

Reply:
you can modify this in next pr